help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Salesforce Pardot] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. 
+
+To correctly map your XDM fields to the [!DNL Salesforce Pardot] destination fields, follow the steps below.
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and specify from the [!DNL Salesforce Pardot API](https://pi.pardot.com/api/v5/objects/prospects/do/upsertLatestByEmail) the required attribute name.
+
+    * Repeat these steps to add any additional mappings between your XDM profile schema and [!DNL Salesforce Pardot]:
+        |Source Field|Target Field| Mandatory|
+        |---|---|---|
+        |`IdentityMap: Email`|`Identity: email`| `Mandatory` |
+
+    * An example with the above mapping is shown below:
+    ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/email-marketing/salesforce-pardot/mappings.png)
+
+When you have finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Navigate to one of the segments you had selected. Then Select the **[!DNL Activation data]** tab, The **[!UICONTROL Mapping ID]** column displays the name of the custom field which is generated within the [!DNL Salesforce Pardot Prospects] page.
```

Comment:
Minor grammar comment: 

~Then~ Select the **[!DNL Activation data]** tab. (period instead of comma)